src/app.js

```diff
@@ -1,8 +1,13 @@
 import express from 'express'
+import fallback from 'express-history-api-fallback'
+import path from 'path'
 import {meta, time_seriesDB, heatmapDB} from './util/initDB.js'
 
 const app = express()
-app.use(express.static('public'))
+const root = path.join(__dirname, '../public')
+console.log(root)
+app.use(express.static(root))
+app.use(fallback('index.html', { root }))
```

Comment:
The call to `/list` is returning HTML. This causes the client-side error as you're trying to tokenise HTML into JSON.

The fallback middleware is being loaded before your endpoint handlers. Make sure you load the endpoints first. One way is to use an Express Router instance to group your API calls into their own module, then import and `use()` that router. This is only an issue because the APIs and static assets are served from the same server. Splitting the backend and frontend into their own apps is common in larger scale apps/teams.

Also note that your fetch request specifies no `Accepts: application/json` header. That makes the fallback middleware think you are happy to accept HTML. If you change the client side API call to ask for the correct MIME type, the problem is also resolved as the fallback will ignore and pass on the request to the endpoint handlers.